ansible/group_vars/all.yml

```diff
@@ -396,6 +215,26 @@ users_deleted:
   - username: glenmailer
```

Comment:
should glen still have access? Wondering since your doing clean up 😅

Reply:
That's the users deleted bit.

Reply:
Im still half asleep 😅